pkg/controller/flinkapplication/flink_state_machine.go

```diff
@@ -22,7 +22,9 @@ import (
 )
 
 const (
-	jobFinalizer = "job.finalizers.flink.k8s.io"
+	jobFinalizer           = "job.finalizers.flink.k8s.io"
+	retryBaseBackoffMillis = 100
+	timeToWaitMillis       = 5 * 60 * 1000
```

Comment:
Should these be configurable? If the timeToWaitMilis is used like the existing staleness duration then it may need to be tuned based on e.g., how fast the cluster can spin up pods.

Reply:
Yeah I think it may make sense to add these two configs to the configmap? Does that make sense?